### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #2: Contributing.md
Please assign this issue to me, I want to create contributing.md to this proj

### PR DESCRIPTION
**Pull request by Devansh-Singh-Parmar**
please add the necessary labels

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Contributing Guidelines
+
+Welcome to our open-source project! By contributing, you help us create better software for everyone.
+
+Before you start contributing, please take a moment to read the following guidelines to ensure a smooth and effective contribution process.
+
+## Technology Stack
+
+![Technology Stack](https://skillicons.dev/icons?i=git,html,css,react)
```
Comment (danger-ahead) [inline CONTRIBUTING.md]: can next logo be added here as well?

Comment (danger-ahead) [reply CONTRIBUTING.md]: Also please remove git from here.

**Comment by Devansh-Singh-Parmar** [comment]
Please add hacktoberfest-accepted label or else merge it.

**Comment by danger-ahead** [comment]
> Please add hacktoberfest-accepted label or else merge it.

Some changes need to be done first. Look at my review.

**Comment by Devansh-Singh-Parmar** [comment]
Thank you for addressing, unfortunately I see no clue of "review" at my end please correct me if i'm wrong.

File: CONTRIBUTING.md (hunk 1 above)
Comment (danger-ahead) [reply CONTRIBUTING.md]: @Devansh-Singh-Parmar here's my review.

**Comment by Devansh-Singh-Parmar** [comment]
Asking for them again
cc @danger-ahead

**Comment by Devansh-Singh-Parmar** [comment]
Done, please check now.

**Review by danger-ahead — APPROVED** [review]

**Comment by Devansh-Singh-Parmar** [comment]
Thank you!

**Pull request merged**